Core/Executor/ContentTypeManager.php

```diff
@@ -284,6 +285,44 @@ protected function delete($step)
     }
 
     /**
+     * Method that create a content type if it doesn't already exist, or update it if it do.
+     */
+    protected function upsert($step)
+    {
+        if (!isset($step->dsl['identifier'])) {
+            throw new \Exception("The 'identifier' key is missing in a content type upsert definition");
+        }
+        if (isset($step->dsl['match'])) {
+            throw new \Exception("The 'match' key is not supported in a content type upsert definition");
+        }
+
+        $contentTypeService = $this->repository->getContentTypeService();
+
+        try {
+            $contentType = $contentTypeService->loadContentTypeByIdentifier($step->dsl['identifier']);
+
+            if (!empty($step->dsl['remove_extra_attributes'])) {
```

Comment:
what about just using existing tag 'remove_attributes' ?
- it already allows to remove specific attributes
- we could add to it support for something like '*', which would mean 'remove all attributes except the ones in the specified list
- this could be done in the 'update' action as well, not only in 'upsert'

Reply:
I changed it to your suggestion 👍